src/android/SplashScreen.java

```diff
@@ -341,7 +341,11 @@ public void onCancel(DialogInterface dialog) {
                     }
                 });
 
-                spinnerDialog.setCancelable(false);
+                if(preferences.getBoolean("spinnerCancelable",true)){
```

Comment:
Should not the property be `false` by default (noncancellable)?
You are changing the default plugin behavior as it is optional.